packages/storage-service/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@bitbucket.org/colbenson/storage-service.git"
+    "url": "git+ssh://git@github.com/empathyco/x.git"
```

Comment:
This should be the url to the repo, without `git+ssh` or `git+https` as specify in the [documentation examples](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository).

Reply:
When the rest of packages are merged, I'll fix all of them.